make/bundle-dist

```diff
@@ -13,29 +13,32 @@ trap "rm -rf '${tmp_dir}'" EXIT
 
 if [[ ${FISSILE_STEMCELL} =~ ^.*fissile-stemcell-opensuse.*$ ]] ; then
     stemcell_os="opensuse"
+    suffix="-opensuse"
 elif [[ ${FISSILE_STEMCELL} =~ ^.*fissile-stemcell-sle.*$ ]] ; then
     stemcell_os="sle"
+    suffix=""
 else
-    echo "Unkown stemcell operating system: ${FISSILE_STEMCELL}"
+    echo "Unknown stemcell operating system: ${FISSILE_STEMCELL}"
     exit 1
 fi
 ARCHIVE_ROOT="${GIT_ROOT}/output/scf-${stemcell_os}-${APP_VERSION}"
 
 echo Packaging, taking "${APP_VERSION}" ...
 
 # Assembling the pieces ...
-mkdir -p ${tmp_dir}/kube ${tmp_dir}/helm
+find "${tmp_dir}"
 
-# kube configs
-tar xf "${GIT_ROOT}/output/${ARTIFACT_NAME}-kube-${APP_VERSION}.tgz" -C "${tmp_dir}/kube"
-# helm charts
-tar xf "${GIT_ROOT}/output/${ARTIFACT_NAME}-helm-${APP_VERSION}.tgz" -C "${tmp_dir}/helm"
+for project in uaa cf ; do
+    mkdir -p "${tmp_dir}/kube/${project}${suffix}"
+    tar xf "${GIT_ROOT}/output/${project}-${stemcell_os}-kube-${APP_VERSION}.tgz" -C "${tmp_dir}/kube/${project}${suffix}/"
+    mkdir -p "${tmp_dir}/helm"
```

Comment:
Do we want to scope this as well to the `$project`? If not, this command can be executed outside the loop.

Reply:
Fixed in bc206b25d8f09322bffb700b7ba72da8783a823c